clang/lib/Tooling/DependencyScanning/DependencyScanningTool.cpp

```diff
@@ -246,16 +248,16 @@ class IncludeTreePPConsumer : public PPIncludeActionsConsumer {
 };
 } // namespace
 
-Error IncludeTreePPConsumer::initialize(CompilerInstance &CI) {
-  if (Error E =
-          PrefixMapping.configurePrefixMapper(CI.getInvocation(), PrefixMapper))
+Error IncludeTreePPConsumer::initialize(CompilerInstance &ScanInstance,
+                                        CompilerInvocation &CI) {
```

Comment:
nit: `CI` is confusing in a context you have both Invocation and Instance, especially it used to reference a different type